is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
+      if (elementToCompare === currentElement) {
+        isAllUnique = false;
+      }
+    });
+  });
+
+  return isAllUnique;
+};
+
```

Comment:
Code is readable -- used good variable names.

Reply:
That's a good comment, I'd suggest using in the summary of your code review, so it's not tagged to an empty line.